tasks/README.md

```diff
@@ -0,0 +1,96 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<end date>",
+  "startedOn":"<start date>",
+  "status": "Active",
+  "ownerId":"<app owner user id>",
+  "percentCompleted":10%,
```

Comment:
We can remove the `%` here. Let's keep it numeric

Reply:
Got it.